app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,10 @@
+module OrdersHelper
+  def order_paid_via(order)
+    # `sort_by` avoids additional database queries when payments are loaded
+    # already. There is usually only one payment and this shouldn't cause
+    # any overhead compared to `order(:updated_at)`.
+    #
+    # Using `last` without sort is not deterministic.
+    order.payments.sort_by(&:updated_at).last.andand.payment_method
```

Comment:
I think this helper method should be called "order_payment_method" and the implementation should call a service (app/services/order_payment) with this code (except the payment_method that can stay here.
I am sure this code will be used in many places in the app. I want to use it already in the subs work I am doing :-)
Maybe I''d call the method "last":
```suggestion
    def order_payment_method
         OrderPayment.new(order).last.payment_method
    end
```

Reply:
I'm happy for you to do the refactoring when you want to use it. You can then see more easily why it has been done that way.

Do you need to find the last payment or also the payment method of the last payment?

Didn't we agree on service names like `OrderPaymentFinder`?

Reply:
yesk, I can do the refactor afterwards.
in StripeSCA it''s mostly fetching the last pending payment.
yes, OrderPaymentFinder is more in accordance with our convention :+1: